app/controllers/uploads_controller.rb

```diff
@@ -125,7 +124,7 @@ def requirements_messages
     custom_validator_messages = "#{klass.name}Validator::REQUIREMENT_DESCRIPTIONS".safe_constantize
     messages.push(*custom_validator_messages)
```

Comment:
These aren't necessary anymore are they?  If you remove them, then you won't be inserting duplicate messages and you won't need to call `.uniq` on `messages` on line 127, right?

Reply:
https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/weam.rb and https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/program.rb need this line

https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/edu_program.rb is the exception which when you viewed this CSV was creating the empty bullet point